crates/edr_provider/src/error.rs

```diff
@@ -305,7 +305,7 @@ impl std::fmt::Display for TransactionFailureWithTraces {
     }
 }
 
-/// Wrapper around [`revm_primitives::Halt`] to convert error messages to match
+/// Wrapper around [`ExecutionResult::Halt`] to convert error messages to match
```

Comment:
I'm not 100% sure about this change.

Reply:
See https://github.com/NomicFoundation/hardhat/pull/5074#discussion_r1551185667